### ISSUE TITLE
[Accessibility]A11y_VSCode_ColorTheme_ColorContrast:Color contrast ratio is less than 4.5:1 for the Black colored content when focused.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [chrmarti/testissues#22790]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.4.3
Tools Used : Color Contrast Analyzer

**Repro Steps:**
1)Navigate to Settings-->Select color theme -->Select light visual studio.
2)Check color contrast ratio for the Explorer content on the screen.

**Actual:**
Color contrast ratio is less than 4.5:1 for the black colored content when focused.
**Similar issue is observed in following themes:**
light Visual studio
Light+(Default light)
Quiet light
Solarized light

**Expected:**
Color contrast ratio should be more than 4.5:1 for the content on a screen.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The low vision/visually impaired users will not be able to view the content properly.

**MAS Reference:**
MAS 1.4.3 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachment For Reference:**
![accessibility a11y_vscode_colortheme explorerblack _colorcontrast focused](https://user-images.githubusercontent.com/39204909/41973909-0ce4ab2a-7a34-11e8-9bca-6f02c38c5356.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by AccessibilityTestingTeam-TCS** [comment]
Unfortunately bug was logged in this path.Hence closing this bug.
